src/cli/mod.rs

```diff
@@ -1,157 +1,39 @@
-/// Arguments consist of Command an Verbosity
-pub mod args;
 /// Commands to run
 pub mod commands;
+/// Command Trait
+mod runnable_command;
 /// Ways of specifying resources
 pub mod specifiers;
 /// Debug level
 pub mod verbosity;
+/// Export
+pub use runnable_command::RunnableCommand;
+
+/// CLI Args
+#[derive(clap::Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+pub struct Args {
+    /// Command passed
+    #[command(subcommand)]
+    pub command: commands::BanyanCommand,
+    /// Verbosity level.
+    #[arg(short, long, help = "verbosity level", default_value = "normal")]
+    pub verbose: verbosity::MyVerbosity,
+}
```

Comment:
I may be just be missing it it but it looks like the verbosity switch isn't wired up to anything

Reply:
nice catch yeah, more vestigial code